crates/proto/src/xfer/dnssec_dns_handle.rs

```diff
@@ -717,15 +717,35 @@ where
                 })
                 .map_ok(|message| {
                     // DNSKEYs were already validated by the inner query in the above lookup
-                    message
+                    let dnskeys = message
```

Comment:
Nit: let's spell this as `dns_keys`?

Reply:
the `dnskey` identifier is used to refers to `DNSKEY` records and we don't hyphenate them as `dns_key` elsewhere in the codebase so I think this pluralization is more consistent with existing usage of the identifier

Reply:
Fair.